README.rst

```diff
@@ -0,0 +1,136 @@
+Syncano command line tool
+=========================
+
+Installation
+------------
+
+::
+
+    pip install syncano-cli
+
+Usage
+-----
+
+First you need to login into your account
+
+::
+
+    syncanocli login
```

Comment:
As far as I remember from sprint planning, we were talking about 'syncano' name :)